applications/simple_pdw_pipeline/metadata.json

```
@@ -0,0 +1,34 @@
+{
+	"application": {
+		"name": "Basic PDW Pipeline",
+		"authors": [
+			{
+				"name": "Joshua Anderson",
+				"affiliation": "GTRI"
+			},
+			{
+				"name": "Christopher Jones",
+				"affiliation": "GTRI"
+			}
+		],
+		"language": "CPP",
```

We are unifying the language to be C++
```suggestion
		"language": "C++",
```